trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorate'
+
+class TrimmerDecorator < BaseDecorate
+  def correct_name
+    @nameable.correct_name[0, 10]
+  end
+end
```

Comment:
optional -
your function should have an if statement which should check the length  of the characters and trim the word 
`
@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
`
👍🏾  💯